stories/Button.stories.tsx

```diff
@@ -31,3 +38,5 @@ export const Small = Template.bind({});
 Small.args = {
   size: "small",
 };
+
+export const AllVariants = Template.bind({});
```

Comment:
chore: I think we should export this as the first story so it opens up by default. Showing large volume of combinations may not be ideal for real projects but captures the essence of the project quickly so best to showcase this first.

Reply:
Great idea. Done!